Test/Utils/ContrailAPI/FloatingIPPool.ps1

```diff
@@ -0,0 +1,36 @@
+. $PSScriptRoot\Constants.ps1
+
+function Add-ContrailFloatingIpPool {
+    Param ([Parameter(Mandatory = $true)] [string] $ContrailUrl,
+           [Parameter(Mandatory = $true)] [string] $AuthToken,
+           [Parameter(Mandatory = $true)] [string] $TenantName,
+           [Parameter(Mandatory = $true)] [string] $NetworkName,
+           [Parameter(Mandatory = $true)] [string] $PoolName)
+
+    $Request = @{
+        "floating-ip-pool" = @{
+            "fq_name" = @("default-domain", $TenantName, $NetworkName, $PoolName)
+            "parent_type" = "virtual-network"
+            "uuid" = $null
+        }
+    }
+
+    $RequestUrl = $ContrailUrl + "/floating-ip-pools"
+    $Response = Invoke-RestMethod `
+        -Uri $RequestUrl `
+        -Headers @{"X-Auth-Token" = $AuthToken} `
+        -Method Post `
+        -ContentType "application/json" `
+        -Body (ConvertTo-Json -Depth $CONVERT_TO_JSON_MAX_DEPTH $Request)
+
+    return $Response.'floating-ip-pool'.'uuid'
+}
+
+function Remove-ContrailFloatingIpPool {
```

Comment:
There are no tests for all these new utility functions.
We can imagine simple integration tests as a good way to proceed.